security-council-rehearsals/2024-01-22-r2-remove-signer-0122/justfile

```diff
@@ -33,7 +33,7 @@ sign hdPath='0':
   echo "Signing with: ${signer}"
   echo ""
   forge build
-  $(go env GOPATH)/bin/eip712sign --ledger --hd-paths "m/44'/60'/{{hdPath}}'/0/0" -- \
+  ./bin/eip712sign --ledger --hd-paths "m/44'/60'/{{hdPath}}'/0/0" -- \
```

Comment:
Should be relative to repository root

Reply:
I think it works because of the `cd ../..` above.

Reply:
The benefit is that you can use a canonical recipe across different folders if you stick with the repo root.

Reply:
could also add:
```
root := git rev-parse --show-toplevel
```

to the header and then just do `$(root)/bin/eip712sign` fwiw.